docs/source/llms/llm-evaluate/notebooks/rag-evaluation.ipynb

```diff
@@ -67,6 +67,26 @@
     "import mlflow"
```

Comment:
nit: This cell appears to be a bit weird place, can we include these import in the cell below?